### PR TITLE
Fix refine plev

### LINKED ISSUE #104: atmosphere.refine_plev is not working
The function atmosphere.refine_plev is returning error messages: 

> d = cls(phlev=dictionary['phlev'])

> KeyError: 'phlev'

### PR DESCRIPTION
**Pull request by lkluft**
This PR fixes #104 and adds a test to check the ``Atmosphere.refine_plev`` interface in the future

### CODE REVIEW

File: konrad/atmosphere.py [hunk 1]
```diff
@@ -240,9 +242,7 @@ def refine_plev(self, pgrid, **kwargs):
                          axis=-1, fill_value='extrapolate', **kwargs)
 
             # Store the interpolated new data in the data directory.
-            # dims = self.default_dimensions[variable]
-            # datadict[variable] = DataArray(f(pgrid), dims=dims)
-            datadict[variable] = f(pgrid).ravel()
+            datadict[variable] = f(plev).ravel()
 
         # Create a new atmosphere object from the filled data directory.
         # This method also calculates the new phlev coordinates.
```
Comment (SallyDa) [inline konrad/atmosphere.py]: Please keep the comments up to date!

Comment (lkluft) [reply konrad/atmosphere.py]: The snippet that I have removed wasn't really a comment, but code that has been commented out. I removed it to ensure that it does not get outdated with time.
Or am I missing something?

Comment (SallyDa) [reply konrad/atmosphere.py]: Sorry I was referring to line 248 (should have made that clear). There are also some later comments that I think are not relevant anymore (regarding atmosphere attributes).

Comment (lkluft) [reply konrad/atmosphere.py]: Ah, that's right! I guess the attributes may still be valid, but to be honest I am not completely sure. Anyhow, I removed line 248 (and pushed directly to master 🙊)

**Pull request merged**